File: Copilot/Java/Facade/round3/source/CarFacade.java
public class CarFacade {
    private Engine engine;
    private Lights lights;
    private SteeringWheel steeringWheel;
    private Brakes brakes;

    public CarFacade() {
        this.engine = new Engine();
        this.lights = new Lights();
        this.steeringWheel = new SteeringWheel();
        this.brakes = new Brakes();
    }

    public void startEngine() {
        engine.start();
    }

    public void stopEngine() {
        engine.stop();
    }

    public void turnOnLights() {
        lights.turnOn();
    }

    public void turnOffLights() {
        lights.turnOff();
    }

    public void turnLeft() {
        steeringWheel.turnLeft();
    }

    public void turnRight() {
        steeringWheel.turnRight();
    }

    public void applyBrakes() {
        brakes.apply();
    }

    public void releaseBrakes() {
        brakes.release();
    }
}

File: Copilot/Python/Facade/round3/source/car_facade.py
class CarFacade:
    def __init__(self):
        self.engine = Engine()
        self.lights = Lights()
        self.steering_wheel = SteeringWheel()
        self.brakes = Brakes()

    def start_engine(self):
        self.engine.start()

    def stop_engine(self):
        self.engine.stop()

    def turn_on_lights(self):
        self.lights.turn_on()

    def turn_off_lights(self):
        self.lights.turn_off()

    def turn_left(self):
        self.steering_wheel.turn_left()

    def turn_right(self):
        self.steering_wheel.turn_right()

    def apply_brakes(self):
        self.brakes.apply()

    def release_brakes(self):
        self.brakes.release()